lang/golang2mgmt/func.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+)
+
+type function struct {
+	MgmtPackage string     `yaml:"mgmtPackage"`
+	MgmtName    string     `yaml:"mgmtName"`
+	Help        string     `yaml:"help"`
+	GoPackage   string     `yaml:"goPackage"`
+	GoFunc      string     `yaml:"goFunc"`
+	Args        []arg      `yaml:"args"`
+	Return      []arg      `yaml:"return"`
+	Tests       []functest `yaml:"tests"`
+}
+
+type functest struct {
+	Args   []testarg `yaml:"args"`
+	Expect []testarg `yaml:"return"`
+}
+
+type templateInput struct {
+	Func        function
+	MgmtPackage string
+}
+
+func parseFuncs(c config, path, templates string) error {
+	templateFiles, err := filepath.Glob(templates)
+	if err != nil {
+		return err
+	}
+	for _, tpl := range templateFiles {
+		log.Printf("Generating %s\n", tpl)
```

Comment:
If the methods aren't on an `obj` which contains a `Logf` method, then you should use the `logf` that you pass down from main. (Here and elsewhere.)

Reply:
Doing this would still be nice :)